dev-tools/generate_notice.py

```diff
@@ -434,6 +434,7 @@ def detect_license_summary(content):
     "MPL-2.0",
     "UPL-1.0",
     "ISC",
+    "UNKNOWN",
```

Comment:
I don't think we can accept these, did you dig in what license is this project using?

Reply:
I did check and `json-patch` has a BSD3 license (which is accepted), but detection is failing, we need to adjust the script to detect it

Reply:
👍 looking into it!

Reply:
@exekias I'm afraid that their licence is not correct:
In https://github.com/evanphx/json-patch/blob/17efcbe3533fdf65efac5054bddbee809b4b4848/LICENSE#L9 there is a comma missing after `notice`.

Cross checked with https://opensource.org/licenses/BSD-3-Clause.

Reply:
Why not, this looks like a minor typo, a PR sounds like a good idea.

In order to avoid blocking this one I wonder if we should update our script to accept that or wait a little bit to see it fixed

Reply:
PR opened. I prefer to have it fixed there properly instead of changing our script :)